src/test/java/exercise/kadai10th/mapper/AirportMapperTest.java

```diff
@@ -1,158 +1,180 @@
 package exercise.kadai10th.mapper;
 
 import exercise.kadai10th.entity.AirportEntity;
-import com.github.database.rider.core.api.configuration.DBUnit;
 import com.github.database.rider.core.api.dataset.DataSet;
 import com.github.database.rider.core.api.dataset.ExpectedDataSet;
 import com.github.database.rider.spring.api.DBRider;
 import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Nested;
 import org.junit.jupiter.api.Test;
 import org.mybatis.spring.boot.test.autoconfigure.MybatisTest;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.boot.test.autoconfigure.jdbc.AutoConfigureTestDatabase;
 import org.springframework.dao.DuplicateKeyException;
 import org.springframework.transaction.annotation.Transactional;
 
-import static com.github.database.rider.core.api.configuration.Orthography.LOWERCASE;
 import static org.assertj.core.api.Assertions.assertThat;
 import static org.assertj.core.api.Assertions.assertThatExceptionOfType;
 import static org.assertj.core.api.Assertions.tuple;
 
 @DBRider
-@DBUnit(caseInsensitiveStrategy = LOWERCASE, schema = "airport_database",
-        url = "jdbc:mysql://localhost:3307/airport_database",
-        user = "user", password = "password")
 @MybatisTest
 @AutoConfigureTestDatabase(replace = AutoConfigureTestDatabase.Replace.NONE)
+@Transactional
 class AirportMapperTest {
 
     @Autowired
-    AirportMapper airportMapper;
-
-    @Test
-    @DataSet(value = "datasets/airports.yml, datasets/prefectures.yml")
-    @Transactional
-    @DisplayName("指定の空港コードが存在するときは、対応する空港EntityをOptionalとして返すこと")
-    void selectAirportByCode() {
-        assertThat(airportMapper.selectAirportByCode("CTS"))
-                .get()
-                .isEqualTo(new AirportEntity("CTS", "新千歳空港", "01", "北海道"));
+    private AirportMapper airportMapper;
+
+    @Nested
+    @DisplayName("Method: selectAirportByCode")
+    class SelectAirportByCodeTest {
```

Comment:
```suggestion
    @Nested
    class SelectAirportByCodeTest {
```
クラス名と書いてあることがほとんど同じなのでDisplayNameなくてもいいのではと思いました。

Reply:
ご指摘の箇所を修正し、若干スッキリました。

![image](https://github.com/SUZUKI-Takayuki-0404/Kadai10th/assets/113277395/73944b89-00cc-4c7f-aec2-f6d23bc85108)